Composer/packages/client/src/pages/design/index.tsx

```diff
@@ -389,6 +389,7 @@ function DesignPage(props) {
                     hidden={triggerButtonVisible || !selected}
                     src={`${rootPath}/extensionContainer.html`}
                     ref={addRef}
+                    title={formatMessage('visual editor')}
```

Comment:
@DesignPolice As a team, do we want to enforce copy rules?
`Visual editor`
also for the changes below: 
`Form editor`,
`Show all notifications`,
`Show ${item} notifications`

Reply:
For reference: https://docs.microsoft.com/en-us/windows/win32/uxguide/text-ui

Reply:
*Looks like it might actually be "Visual Editor". I remember the rules incorrectly :/

Reply:
Thanks @corinagum - I think we should be following any and all copy rules. My guess is that we have a mix of going to fast and not one source for copy... which we should have. I will tag @mareekuh on this and maybe we can get Yan to take a pass, unless you are wanting the practice... but, I think copy probably needs some help outside of just the rules.

Reply:
Either way, happy to help out. lmk!